Classes/Utility/FLEXHeapEnumerator.m

```diff
@@ -71,8 +70,33 @@ + (void)enumerateLiveObjectsUsingBlock:(flex_object_enumeration_block_t)block
     if (result == KERN_SUCCESS) {
         for (unsigned int i = 0; i < zoneCount; i++) {
             malloc_zone_t *zone = (malloc_zone_t *)zones[i];
-            if (zone->introspect && zone->introspect->enumerator) {
-                zone->introspect->enumerator(TASK_NULL, (__bridge void *)block, MALLOC_PTR_IN_USE_RANGE_TYPE, (vm_address_t)zone, reader, &range_callback);
+            malloc_introspection_t *introspection = zone->introspect;
+
+            if (!introspection) {
+                break;
+            }
+
+            void (*lock_zone)(malloc_zone_t *zone)   = introspection->force_lock;
+            void (*unlock_zone)(malloc_zone_t *zone) = introspection->force_unlock;
+
+            // Callback has to unlock the zone so we freely allocate memory inside the given block
```

Comment:
It would seem not! I can't find an app in which it still crashes. I'll ask around on /r/jailbreakdevelopers and /r/jailbreak to see if anyone can find crashes.